.github/workflows/ci.yml

```diff
@@ -87,8 +87,8 @@ jobs:
           git config --local user.email "rushbot@users.noreply.github.com"
         working-directory: repo-b
 
-      - name: Rush install (rush-lib)
-        run: node ${{ github.workspace }}/repo-a/apps/rush/lib/start-dev.js install
+      - name: Rush update (rush-lib)
+        run: node ${{ github.workspace }}/repo-a/apps/rush/lib/start-dev.js update
```

Comment:
This shouldn't behave any differently than `rush install` here.

Reply:
No. Here, the Rush command is using the `rush-lib` version, not the Rush version defined in the `rush.json`.
So, it is possible that the lockfile is out of date for `rush-lib` Rush version, that's why we need `rush update` to update it.

Reply:
During CI, those changes don't get checked back in.

Is the lockfile currently out-of-date, as of this PR?

Reply:
@iclanton this is the same problem that we identified previously, where the CI pipeline is trying to run "rush install" using a different Rush version than `rushVersion` from **rush.json**.  Generally that is NOT expected to work.

`rush update` is an okay workaround I think.

Longer term, what we should do is perform this E2E test using a separate testing repository, rather than trying to run it against Rush Stack's own repository.  That way the separate testing repository can run `rush update` (using the local build) and  commit the updates to Git, which would eliminate this inconsistency and actually provide a more realistic test.

@dmichon-msft